docs/run.rst

```diff
@@ -59,6 +59,9 @@ Options
 ``-D --dry_run``
     Run without injecting payments. Suitable for testing. Does not require locking.
```

Comment:
Idea: Move option "Dp" up to group all "dry-run-options". Create a small glossary like p=no publish stats and s=no signer and make clear that there are possibilities how those options can be combined.